pkgs/tools/inputmethods/fcitx5/with-addons.nix

```
@@ -11,7 +13,8 @@ symlinkJoin {
     wrapProgram $out/bin/fcitx5 \
       --prefix FCITX_ADDON_DIRS : "$out/lib/fcitx5" \
       --suffix XDG_DATA_DIRS : "$out/share" \
-      --suffix PATH : "$out/bin"
+      --suffix PATH : "$out/bin" \
+      --suffix LD_LIBRARY_PATH : ${makeLibraryPath [fcitx5-lua.lua]}
```

```suggestion
      --suffix LD_LIBRARY_PATH : ${makeLibraryPath (lib.flatten (map (x: x.passthru.extraLdLibraries) addons)) }
```

Or

```suggestion
      --suffix LD_LIBRARY_PATH : ${makeLibraryPath (lib.flatten (map (x: x.propogratedBuildInputs) addons)) }
```

You can do something like this which can work for other similar issues besides lua.